core/src/main/scala/org/apache/spark/shuffle/IndexShuffleBlockResolver.scala

```diff
@@ -75,13 +81,68 @@ private[spark] class IndexShuffleBlockResolver(conf: SparkConf) extends ShuffleB
   }
 
   /**
+   * Check whether there are index file and data file also they are matched with each other, returns
+   * the lengths of each block in data file, if there are matched, or return null.
```

Comment:
```
/**
 * Check whether the given index and data files match each other.
 * If so, return the length of each block in the data file, otherwise return null.
 */
```